flink-table/flink-sql-client/src/main/java/org/apache/flink/table/client/SqlClient.java

```diff
@@ -90,6 +96,7 @@ private void start() {
 
 			// create CLI client with session environment
 			final Environment sessionEnv = readSessionEnvironment(options.getEnvironment());
+			appendPythonConfig(sessionEnv, options.getPythonConfiguration());
```

Comment:
could use the `enrich` method defined in Environment as following:
`
final Environment sessionEnvWithPythonConfiguration = Environment.enrich(
				sessionEnv, options.getPythonConfiguration().toMap(), Collections.emptyMap());
`

Reply:
It seems the `enrich()` method of the `ConfigurationEntry` only accepts the configurations that start with "table.". As the python configurations all start with "python.", the `enrich` method does not work for python configurations.